crates/llm-base/src/lora.rs

```diff
@@ -126,7 +126,10 @@ impl LoraAdapter {
 
         // Compute the graph
         gf.build_forward_expand(&output);
-        patch_context.graph_compute(&mut gf);
+
+        //TODO: maybe pass the model's thread count to this context
+        let mut plan = GraphExecutionPlan::new(&mut gf, 8);
```

Comment:
Yes, I'd like to resolve the matter of implicit thread counts for non-inference operations. I wonder what the best approach would be there - maybe we should just add a `ExecutionParameters { thread_count, backend }` struct as a parameter to each function? 🤔

Reply:
Good idea but i probably wont implement this in this PR 👍